lib/bolt/module_installer.rb

```diff
@@ -144,8 +149,8 @@ def install_puppetfile(path, moduledir, config = {})
         raise Bolt::Error.new(
```

Comment:
Ugh, this error was updated as part of #2168 , but wasn't carried over in #2134 . Would you mind adding it back in here? https://github.com/puppetlabs/bolt/commit/b0db40014e28b4f07f78392554619dd894dba664#diff-b4ccc59dee0e054966d665d51e08d67eR895-R903